src/selection.js

```diff
@@ -53,22 +53,42 @@ function Selection(currentElem, nativeSelection) {
     }
   };
 
+  me.getParentElement = function() {
```

Comment:
This seems to assume that the selection is all within a single parent, what if the selection is like:
`<p><b>12[34</b>as]df</p>`
Should the parent element be the `<b>` element or the `<p>`?